src/ar_http_iface_middleware.erl

```diff
@@ -460,6 +460,22 @@ handle(<<"GET">>, [<<"wallet">>, Addr, <<"last_tx">>], Req) ->
 			Req}
 	end;
 
+%% @doc Return a block anchor to use for building transactions.
+handle(<<"GET">>, [<<"tx_anchor">>], Req) ->
+	case ar_node:get_hash_list(whereis(http_entrypoint_node)) of
+		[] ->
+			{400, #{}, <<"The node has not joined the network yet.">>, Req};
+		BHL when is_list(BHL) ->
+			{
+				200,
+				#{},
+				ar_util:encode(
+					lists:nth(min(length(BHL), (?MAX_TX_ANCHOR_DEPTH)) div 2 + 1, BHL)
```

Comment:
So in the end, we always pick the block exactly in the middle of the allowed range